src/main/java/blusunrize/immersiveengineering/client/render/tile/ShaderBannerRenderer.java

```diff
@@ -37,7 +38,7 @@ public void render(ShaderBannerTileEntity te, double x, double y, double z, floa
 	{
 		long time = te.getWorldNonnull().getGameTime();
 		GlStateManager.pushMatrix();
-		if(!te.wall)
+		if(te.getState().getBlock() == Cloth.shaderBanner)
```

Comment:
Throw a comment here to make it clear what that condition does, please.
`wall` was self-explanatory, this one is a little more obscure.